components/typography/Paragraph.stories.tsx

```diff
@@ -0,0 +1,41 @@
+import type { Meta, StoryObj } from '@storybook/react';
```

Comment:
Why there are no warnings for ESLint in these files for adding Jsdoc comments?

Reply:
I didn't get this point, I haven't added any JSDoc in this file.